### DISCUSSION TITLE
nvbootctl absent from image

### DISCUSSION THREAD

**Discussion by formidable01** [OE4T/meta-tegra#1504] [General]
I am building the 'core-image-minimal' from the nanbield branch targeting an Orin AGX. I set USE_REDUNDANT_FLASH_LAYOUT_DEFAULT = "1" in my local.conf  I was hoping to try out the A/B feature and monitor operation using nvbootctrl but that application does not appear to be in the image.  Is there some specific variable or package I might be missing?

**Comment by dwalkes** [comment]
`nvbootctrl` is in tegra-redundant-boot-base I believe, which I think should be in a MACHINE RDEPENDS.  I haven't tested with core-image-minimal though.  You might want to try with tegra-demo-distro and one of the demo images if you find problems with the minimal images.

**Reply by formidable01** [reply]
I'll look into the tegra-demo-distro.  Thanks!
